### ISSUE TITLE
Search while typing title on new post to minimize duplicate threads

### ISSUE THREAD

**Issue by slrgt** [LemmyNet/lemmy#3798]
### Requirements

- [X] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a feature request? Do not put multiple feature requests in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Is your proposal related to a problem?

Many many duplicate threads are made in the same community because the search function is so hidden away.

### Describe the solution you'd like.

while typing a title when creating a new post it should search and display similar threads titles so you can click them and see if a thread already answers your question etc. 

### Describe alternatives you've considered.

Forum software like discourse minimize Duplicate threads by implementing the search in the title section of the workflow of creating a new post

**Comment by lionirdeadman** [comment]
If only title, it might be worthwhile to limit that to a certain amount of time to avoid necroposting.

**Comment by MV-GH** [comment]
Sounds more like a lemmy-ui UX issue, or is it not possible to search for titles in a community?

**Comment by lionirdeadman** [comment]
I don't believe it is possible to search a query *only in titles* so it would also include query results for what's in the description or comments, I think

**Comment by slrgt** [comment]
> I don't believe it is possible to search a query _only in titles_ so it would also include query results for what's in the description or comments, I think

thats fine, the point is to search while typing in the title of a new post so you see similar threads and you don’t make a duplicate thread if one already exists that discusses what you need

**Comment by lionirdeadman** [comment]
In that case, if that's what you want, I'll transfer to ui.
